templates/layout/base.html

```diff
@@ -25,6 +25,16 @@
         </script>
         <script async src="{{cdn 'assets/dist/theme-bundle.head_async.js'}}"></script>
 
+        <script src="https://ajax.googleapis.com/ajax/libs/webfont/1.6.26/webfont.js"></script>
+
+        <script>
+            WebFont.load({
+                custom: {
+                families: ['Karla', 'Roboto', 'Source Sans Pro']
```

Comment:
Could you please explain how you defined list of the fonts?

Reply:
This fonts are variants we have to setup
<img width="1147" alt="Screenshot 2020-10-12 at 11 37 38" src="https://user-images.githubusercontent.com/66325265/95724783-777d1c80-0c7f-11eb-88dd-b991dbe1f70c.png">

Reply:
1. Would be nice if we could find a way how to load only relevant font
2. While loading page Monsterrat font is fetching
<img width="808" alt="Screenshot 2020-10-12 at 14 54 32" src="https://user-images.githubusercontent.com/66319629/95743782-f469bf80-0c9a-11eb-8ae0-9318fe8b9beb.png">
3. From theme setting we can see Arial font at least
<img width="968" alt="Screenshot 2020-10-12 at 14 56 59" src="https://user-images.githubusercontent.com/66319629/95743927-33981080-0c9b-11eb-840c-681b0e361565.png">

Reply:
How many requests do we send for fetching fonts after your changes?

Reply:
> How many requests do we send for fetching fonts after your changes?

3 font requests - as it was